suite-native/graph/src/components/GraphFiatBalance.tsx

```diff
@@ -37,13 +46,13 @@ export const GraphFiatBalance = ({
     const firstGraphPoint = useAtomValue(referencePointAtom);
 
     if (!firstGraphPoint) {
-        return <BoxSkeleton width={120} height={73} />;
+        return <Skeleton />;
     }
 
     return (
-        <>
+        <Box alignItems="center" style={{ height: 72 }}>
```

Comment:
Avoid inline styles, please. 

And also - is the fixed height here really necessary? It's kind of ok for the skeletons, but here it's container for real values and if user have bigger font stale, the value can't fit here, I guess

Reply:
I don't like it here either but I don't know how to ensure that the height doesn't change after the loading is done. That's what causes the layout to shift. 😕 

I'm open to other ideas if you have some.

Reply:
Never mind, it looks OK even on the biggest scale, so let's not spend more time on it now
<img width="377" alt="image" src="https://github.com/user-attachments/assets/a6fffda9-1065-456c-8386-2240378d8eaf">